spec/calendar.html

```diff
@@ -1025,8 +1025,9 @@ <h1>Temporal.Calendar.prototype.inLeapYear ( _dateOrDateTime_ )</h1>
         1. Let _calendar_ be the *this* value.
         1. Perform ? RequireInternalSlot(_calendar_, [[InitializedTemporalCalendar]]).
         1. Assert: _calendar_.[[Identifier]] is *"iso8601"*.
-        1. Let _year_ be ? ISOYear(_dateOrDateTime_).
-        1. Return ! IsISOLeapYear(_year_).
+        1. If Type(_temporalDateLike_) is not Object or _temporalDateLike_ does not have an [[InitializedTemporalDate]] or [[InitializedTemporalYearMonth]] internal slot, then
+          1. Set _temporalDateLike_ to ? ToTemporalDate(_temporalDateLike_).
+        1. Return ! IsISOLeapYear(_temporalDateLike_.[[ISOYear]]).
```

Comment:
same here, if the other similar changes are made:
```suggestion
          1. Return ! IsISOLeapYear(? ToTemporalDate(_temporalDateLike_)).
        1. Return ! IsISOLeapYear(_temporalDateLike_.[[ISOYear]]).
```